staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go

```diff
@@ -133,17 +133,23 @@ func (msg *Message) GetTimestamp() int64 {
 }
 
 // GetContent returns message content
-func (msg *Message) GetContent() interface{} {
+func (msg *Message) GetContent() Content {
+	if msg.Content == nil {
+		return rawContent{}
+	}
 	return msg.Content
 }
 
 // GetContentData returns message content data
 func (msg *Message) GetContentData() ([]byte, error) {
-	if data, ok := msg.Content.([]byte); ok {
+	content := msg.Content
+	if content == nil {
+		return []byte("null"), nil
```

Comment:
```suggestion
	return nil, fmt.Errorf("the content of the message is empty")
```

Reply:
https://github.com/kubeedge/kubeedge/pull/2753#discussion_r614657588
@GsssC What do you think